mpdev/internal/apply/types.go

```diff
@@ -0,0 +1,47 @@
+package apply
+
+
+const API_VERSION = "dev.marketplace.cloud.google.com/v1alpha1"
+var TypeMapper = map[TypeMeta]func() Resource{
+	{ ApiVersion: API_VERSION, Kind: "GceImage" } :             func() Resource { return &GceImage{} },
+	{ ApiVersion: API_VERSION, Kind: "PackerGceImageBuilder"} : func() Resource { return &PackerGceImageBuilder{} },
+}
+
+type TypeMeta struct {
```

Comment:
Did you look into whether we can use the core k8s libraries? OK to follow up.

Reply:
I copied these definitions from k8s libraries, thought I would see about using them directly later